src/cigar.py

```diff
@@ -73,5 +78,10 @@ def edits_to_cigar(edits: str) -> str:
     '1M1D6M1I4M'
 
     """
-    # FIXME: Compute the cigar
-    return ''
+    element_list = split_blocks(edits)
+    cigar=''
+    for a in element_list:
+        short = '{}{}'.format(str(len(a)),str(a)[0])
```

Comment:
If `a` is already a string (and it should be here), then writing `str(a)` is just a long way of writing `a`. You get `a` back when you try to turn a string into a string. So `str(a)[0]` could be written as just `a[0]`.

When you format a string, Python should handle the type conversion automatically, so `str(len(a))` could just be `len(a)`.

Lastly, Python has formatting strings, and has had them for a while, so you could write this line succinctly as 

```python
    short = f"{len(a)}{a[0]}"
```

which I find somewhat easier to read (but YMMV).

You could even go overboard and do the entire loop as a generator expression combined with a join:

```python
cigar = "".join( f"{len(a)}{a[0]}" for a in split_blocks(edits) )
```

but that might be a bit too succinct. I'm not sure. It looks like the kind of code I would write, but that doesn't mean that it is readable to others.